app/controllers/api_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
   private
 
   def entrypoint_versions
-    version_config[:definitions].select(&:ident).collect do |version_specification|
+    Api::Settings.version[:definitions].select(&:ident).collect do |version_specification|
```

Comment:
inconsistent/confusing calling this way, some Settings, some Api::Settings,  in which case i prefer version_config